traces-observer-service/README.MD

```
@@ -0,0 +1,199 @@
+
+Build the image:
+
+```bash
+docker build -t traces-observer-service .
+```
+
+Run the container (example):
+
+```bash
+docker run --rm -p 9098:9098 \
+	-e OPENSEARCH_URL=http://host.docker.internal:9200 \
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)